app/web_framework/cluster_master_application.py

```diff
@@ -248,14 +260,16 @@ class _BuildResultHandler(ClusterBaseHandler, tornado.web.StaticFileHandler):
 
     From the Tornado docs: "for heavy traffic it will be more efficient to use a dedicated static file server".
     """
-    def initialize(self, route_node=None, cluster_master=None):
+    def initialize(self, route_node=None, cluster_master=None, session_id=None):
```

Comment:
This API endpoint failed in the end-to-end tests before I added the session_id parameter. It looks like the whole reason why this customization exists is to set the 'path' parameter to None in the super method.